src/idsk/components/table/template.njk

```diff
@@ -0,0 +1,110 @@
+{% from "../button/macro.njk" import idskButton %}
+<div data-module="idsk-table" id="{{ params.uniqueId }}">
+  <div class="idsk-table__heading">
+    <div class="idsk-table__heading-with-caption">
```

Comment:
- tato classa vyzera ze sa v scss ani nepouziva, neviem ci ma vyznam.
- plus mozno by trebalo pred div-ko vybrat if (title or caption) lebo ak nebude ani jedno ani druhe bude zbytocne prazdne divko

Reply:
plus su tu nejake merge confilicty ta este pls prve pullni dev. Najskor asi tie yaml refactor veci to sposobili